### ISSUE TITLE
[ktx] CMake configure error

### ISSUE THREAD

**Issue by liuhaowen478** [microsoft/vcpkg#18556]
**Describe the bug**

**Environment**
- OS: Windows 10 Build 21390
- Compiler: Visual C++ 2019

**To Reproduce**
1. ./vcpkg install ktx:x64-windows
2. In a Visual Studio CMake project,
```
find_package(Ktx CONFIG REQUIRED)
target_link_libraries(main PRIVATE KTX::ktx)
```
3. Project ->Generate cache

**Expected behavior**
CMake configuration should succeed.

**Failure logs** 
CMake outputs:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error		CMake Error at xxx/vcpkg/installed/x64-windows/share/ktx/KtxTargets.cmake:80 (message):
  The imported target "KTX::ktx" references the file

     "xxx/vcpkg/installed/x64-windows/debug/bin/ktx.dll"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "xxx/vcpkg/installed/x64-windows/share/ktx/KtxTargets.cmake"

  but not all the files it references.		xxx/vcpkg/installed/x64-windows/share/ktx/KtxTargets.cmake	80	

```

**Comment by FrankXie05** [comment]
@liuhaowen478 Sorry for the late reply, Could you please provide your project files? 

- CMakeLists.txt
- vcpkg.json

**Comment by dg0yt** [comment]
Already fixed in #23942.
